.github/workflows/e2e-rocketmq.yaml

```diff
@@ -28,8 +28,6 @@ jobs:
     strategy:
       matrix:
         case:
-          - name: Pulsar
-            path: test/e2e/case/pulsar/e2e.yaml
           - name: Rocketmq
             path: test/e2e/case/rocketmq/e2e.yml
```

Comment:
Why do we need another file for this?

Reply:
My local test runs well, and there are no obvious errors read from the online log. So I consulted Mr. Zhuang
@dmsolr He suggested me to do so.

Reply:
OK, let's see. There is no point this control file relating to this.

Reply:
they are building provider and consumer images by self on one VM. if they are same name in compose file, they will get failed.
another, they are two unrelated tasks.

Reply:
Thank you sir. The online test still failed. I'll check my code and local experiments again.

Reply:
OK, if they don't share image, then it makes sense.

Just one more question, @dmsolr, how are we going to release on this repo? It seems we are going to have 2 OAP-extension releases?

Reply:
> The online test still failed

From the logs, look like the container of RocketMQ broker is not up and running.

Reply:
I perfer to release it separately if we can do.

Reply:
In this repo, I think we can release the packages in any way we prefer.

If doing that, we need 2 plugin jars and 2 OAP-extension packages. We need a makrfile for that, right?